packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1294,22 +1388,58 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   ///    sheet.
   ///  * [Scaffold.of], for information about how to obtain the [ScaffoldState].
   ///  * <https://material.google.com/components/bottom-sheets.html#bottom-sheets-persistent-bottom-sheets>
-  PersistentBottomSheetController<T> showBottomSheet<T>(WidgetBuilder builder) {
+  StandardBottomSheetController<T> showBottomSheet<T>(
+      WidgetBuilder builder, {
```

Comment:
[per our discussion] It would be nice if initialTop was expressed as a percentage of the screen's height.

Reply:
This is done in 2af5a78